hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java

```diff
@@ -235,13 +235,22 @@
             = new HazelcastProperty("hazelcast.operation.call.timeout.millis", 60000, MILLISECONDS);
 
     /**
-     * If an operation has backups and the backups don't complete in time, then some cleanup logic can be executed.
-     * This property specifies that timeout for backups to complete.
+     * If an operation has backups, this property specifies how long the invocation will wait for acks from the backup replicas.
+     * If acks are not received from some backups, there will not be any rollback on other successful replicas.
      */
     public static final HazelcastProperty OPERATION_BACKUP_TIMEOUT_MILLIS
             = new HazelcastProperty("hazelcast.operation.backup.timeout.millis", 5000, MILLISECONDS);
 
     /**
+     * When this configuration is enabled, if an operation has sync backups and acks are not received from backup replicas
+     * in time, or the member which owns primary replica of the target partition leaves the cluster, then the invocation fails
+     * with {@link com.hazelcast.core.OperationBackupAckTimeoutException}. However, even if the invocation fails,
+     * there will not be any rollback on other successful replicas.
+     */
+    public static final HazelcastProperty FAIL_ON_OPERATION_BACKUP_TIMEOUT
+            = new HazelcastProperty("hazelcast.operation.backup.fail.on.timeout", false);
```

Comment:
Perhaps the following is more readable

hazelcast.operation.fail.on.backup.timeout